AT/ad_thread/ad_routine.py

```diff
@@ -0,0 +1,138 @@
+import time
+from statsmodels.tsa import ar_model
+
+
+def compute_zone(value, info):
+    """
+    This function returns an integer depending on the threshold zone on which a value is located.
+    """
+
+    zone = 2
+    if value <= info['low_critic_threshold']:
+        zone = -2
+    elif value <= info['low_warning_threshold']:
+        zone = -1
+    elif value <= info['high_warning_threshold']:
+        zone = 0
+    elif value <= info['high_critic_threshold']:
+        zone = 1
+
+    return zone
+
+
+def threshold_check(value, info):
+    """
+    This function returns an string depending on the threshold zone on which a value is located.
+    """
+
+    message = ''
+    zone = compute_zone(value, info)
+    if zone == -2:
+        message = 'below low critical threshold'
+    elif zone == -1:
+        message = 'below low warning threshold'
+    elif zone == 0:
+        message = ''
+    elif zone == 1:
+        message = 'above high warning threshold'
+    elif zone == 2:
+        message = 'above high critical threshold'
+
```

Comment:
Raise an exception if zone is not a valid value (https://docs.python.org/3/tutorial/errors.html)

Reply:
Even if the code prevents that from happening (for sure)?